src/lavinmqperf.cr

```diff
@@ -264,7 +264,7 @@ class Throughput < Perf
         ch.queue(@queue, passive: true)
       end
       ch.tx_select if @ack_in_transaction > 0
-      ch.prefetch @prefetch unless @prefetch.zero?
+      ch.prefetch @prefetch unless @prefetch == UInt16::MAX
```

Comment:
Maybe skip this as it would yield different prefetch values for lavin and rabbit
```suggestion
```